vmdk_plugin/drivers/photon/photon_driver.go

```diff
@@ -113,25 +113,25 @@ func NewVolumeDriver(targetURL string, projectID string, hostID string, mountDir
 // map hasn't been initialized, return 1 prevent detach and remove.
 
 // Return the number of references for the given volume
-func (d *VolumeDriver) getRefCount(vol string) uint {
+func (d *VolumeDriver) getRefCount(vol string) (uint, error) {
```

Comment:
Fine to me. We can have a small fix for now to avoid possible regressions, and we can refactor/improve this part in the future if needed.